hooks/post_gen_project.py

```diff
@@ -0,0 +1,17 @@
+from shutil import copyfile
+from os import listdir
+
+base_path_libs = '../libs'
+
+alldir = listdir(base_path_libs)
+
+libs = list(filter(lambda x: x.endswith('.jar'), alldir))
+
+for lib in libs:
+    path_lib = base_path_libs +'/'+ lib
+    destiny_lib = './libs/' + lib
```

Comment:
It's best to use `os.path.join` here to join the paths using the directory separator of each platform (e.g. on Windows, it's `\` instead of `/`).
I recommend using the pattern of importing `os` at the top and using the functions as `os.FUNCTION`/`os.path.FUNCTION`.

Small English nitpicking: you probably want to use `target` or `destination` instead of `destiny` (I know, same word in Portuguese, it's confusing). =)